### ISSUE TITLE
Show slug with prefix in graphql

### ISSUE THREAD

**Issue by ishigami** [dimitrov-adrian/directus-extension-wpslug-interface#7]
Hi @dimitrov-adrian 

First of all. Thanks for this extension. It helps a lot.

I have a question, but I don't know if it is a issue.

I'm working with graphql to make the queries in my collections.

And I have a collection named Projects. An this collection have a slug field, with a prefix defined (project-). i've put this prefix in the Interface and the Display sections.

But when I get the slug field through the graphql query, the slug cames without the prefix.

Is it the normal behavior or is it an issue?

Thanks again for your time.

**Comment by dimitrov-adrian** [comment]
Yes, actually this is the correct behaviour. I am using prefix/suffix mostly for the external link building and in app they are handled independently.

Option for storing them as well will be quite ease to add, just I am not sure how usable this will be.
